cmd/kubeadm/app/api/types.go

```diff
@@ -71,6 +71,8 @@ const (
 	DefaultServiceDNSDomain   = "cluster.local"
 	DefaultServicesCIDRString = "100.64.0.0/12" // Carrier-grade NAT range (RFC 6598)
 	DefaultKubernetesVersion  = "v1.4.0-beta.10"
+	DefaultCloudConfigPath    = "/etc/kubernetes/cloud-config.json"
+	KubeadmVersion            = "v0.1.0"
```

Comment:
Can you describe exactly where the v0.1.0 `kubeadm` version number will appear versus the `1.5.0-alpha` that will appear in the package version for `kubeadm`? I'm concerned that showing two different version numbers will be potentially confusing for users. I'd prefer to help users differentiate between kubeadm releases with date- or sha1-based versioning for now, at least until we get a chance to discuss this in the SIG.